docs/source/migrations/1.0.mdx

```diff
@@ -6,69 +6,69 @@ description: From 0.x to 1.0
+
+Apollo iOS 1.0 expands on this, generating an _entire module_ that contains models and metadata for your GraphQL schema and its type definitions.
 
-Apollo iOS 1.0 expands on this, generating a whole module that contains models and metadata for your GraphQL schema and its type definitions.
 In addition to your input objects and enum types, this module contains:
-- Types for the objects, interfaces, and unions referenced
-- Editable definitions for your custom scalars
-- An editable `SchemaConfiguration.swift` file
-- Metadata used by the Apollo GraphQL executor
+- Types for the objects, interfaces, and unions in your schema.
+- Editable definitions for your custom scalars.
+- An editable `SchemaConfiguration.swift` file.
+- Metadata for the Apollo GraphQL executor.
 
-To prevent naming conflicts with other types in your project, the schema types are contained in their own namespace.
-This namespace can be generated as a stand-alone module, which can be imported by your project, or as a caseless namespace enum which can be embedded in your application target.
+Schema types are contained in their own namespace to prevent naming conflicts. You can generate this namespace as a stand-alone module, imported by your project, or as a caseless namespace enum that you can embed in your target application.
 
 ### Multi-module support
 
-Apollo iOS 1.0 was designed to support complex applications composed of multiple modules as well as monolithic application targets.
-
-To support multi-module projects, the [generated schema module](#generated-schema-module) contains all of the schema's shared types and metadata. This allows your generated operation models to be located anywhere you would like in your project structure, as long as they are linked to the schema module.
+Apollo iOS 1.0 can support complex applications composed of multiple modules _and_ monolithic application targets.
 
-The code generation engine provides flexible configuration options that aims to make code generation work seamlessly for any project structure you prefer.
+The [generated schema module](#generated-schema-module) supports multi-module projects by containing all of a schema's shared types and metadata. This enables you to move generated operation models anywhere within your project structure (as long as they are linked to the schema module).
 
+Apollo iOS's code generation engine provides flexible configuration options that make code generation work seamlessly for any project structure.
 
-## Step-by-step instructions
+## Migration walk-through
 
-> **Before performing the migration to Apollo iOS 1.0, you should consider your project structure and decide how you would like to include your generated schema module and operation models.**
+> **Before migrating to Apollo iOS 1.0, consider your project structure and how you want to include your generated schema module and operation models.**
```

Comment:
I don't like this new wording. The old wording may be a little verbose, but it is actually pretty important that people do this, and I feel like this sentence doesn't make it clear how imperative it is that people take a minute to consider their project's organization before starting on the migration.

Not understanding how you want to structure your project (and configuring code generation accordingly) has been the #1 cause of confusion (and GitHub issues) since 1.0 released.